### DISCUSSION TITLE
Counter members

### DISCUSSION THREAD

**Discussion by dennisangemi** [opendatasicilia/opendatasicilia.it#67] [Domande] [answered]
Buongiorno, buon pomeriggio, buonasera, buona notte a tutti/e!
Mi chiedevo se fosse possibile "_chiamare_" in _qualche modo_ dati del tipo:
- numero membri gruppo telegram
- numeri membri gruppo facebook
- numeri iscritti in ml

Non ho ancora un'idea chiara su come e dove disporli nel nuovo sito. Chiedo solo per farmi un'idea ;)

**Comment by gabacode** [comment, marked as answer]
Si, possiamo farlo e mi sembra un'idea carina!
Appunti:
- numero membri gruppo telegram ([serve](https://core.telegram.org/bots/api#getchatmembercount) un bot, api key e group_id (potremmo usare dasoe_report_bot o crearne un altro))
- numeri membri gruppo facebook ([serve](https://developers.facebook.com/docs/graph-api/reference/v12.0/group/opted_in_members) {group-id} e probabilmente App-review)
- numero iscritti in ml ([serve](https://developers.google.com/admin-sdk/directory/reference/rest/v1/members/list) {group_key})
